.devcontainer/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM mysql:5.7
 LABEL maintainer="Pieter Verschaffelt <pieter.verschaffelt@ugent.be>"
 
 RUN mkdir -p /usr/share/man/man1
-RUN apt-get update && apt-get install -y default-jdk git dos2unix maven make wget unzip expect gawk binutils gcc libssl-dev
+RUN apt-get update && apt-get install -y default-jdk git dos2unix maven make wget unzip expect gawk binutils gcc libssl-dev uuid-runtime pv
```

Comment:
Add node as an additional dependency? (Also remove make, I forgot to remove it in the PR where I removed the dependency on make.)

Reply:
* make is still required for building curl ourselves (this is required due to a bug in the curl-version that's bundled with the mysql-docker)
* I will add Node as a dependency :)